package.json

```diff
@@ -77,6 +77,7 @@
     "rollup-plugin-babel": "^3.0.7",
     "rollup-plugin-commonjs": "^9.1.5",
     "rollup-plugin-node-resolve": "^3.3.0",
+    "rollup-plugin-re": "^1.0.7",
```

Comment:
Should we remove gulp-rename?  Is that being used elsewhere?

Reply:
It's used in the gulpfiles for icon, layouts, pfelement, and cp-theme.